2/scss/about-skout.scss

```diff
@@ -0,0 +1,73 @@
+@import "vars";
+@import "fonts";
+@import "mixins";
+
+.about-skout-container {
+  max-width: 1440px;
+  margin: auto 20px;
+  display: flex;
+  flex-flow: wrap;
+  justify-content: center;
+  flex-direction: column;
+  .skout-sa {
+    order: 2;
+    opacity: 0;
+    transition: opacity 700ms;
+    .skout-sa-img{
+      width: 100%;
+    }
+  }
+  .about-skout {
+    order: 1;
+    opacity: 0;
+    transition: opacity 700ms;
+    max-width: 560px;
+    margin: auto;
+    .title-about-skout {
+      margin: 30px 0 20px 0;
+      @include text-size-standart;
+    }
+    .text-about-skout {
+      margin: auto;
+      .text-about-skout-value{
+        color: $colorTextPrimaryDark;
+        font-family: Aclonica;
+        @include text-size(14px, 24px);
+      }
+    }
+  }
+  .buttons-about-skout {
```

Comment:
could be `.about-skout-buttons` and move inside `.about-skout` block of styles `&-buttons`